connectors/google/google-sheets/element-templates/google-sheets-outbound-connector.json

```diff
@@ -443,6 +443,12 @@
     "label" : "Row index",
     "description" : "Enter row index. Details in the <a href=\"https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/google-sheets/#what-is-a-row-index\" target=\"_blank\">documentation</a>",
     "optional" : true,
+    "constraints" : {
+      "notEmpty" : false,
+      "pattern" : {
+        "value" : "^(=.*|[0-9].*|)$"
```

Comment:
I think it's not correct, `9abc` would be a valid input using this. Maybe should you change the digit part to [0-9]+ or \d+